elastic/kibana
# [App Search] Add Curations overview table

## Summary

Adds the Curations overview (mostly just a table) view.

## Screenshots

Curations table:
<img width="1184" alt="" src="https://user-images.githubusercontent.com/549407/108121397-9b9aa100-7057-11eb-8510-fca7e743506b.png">

Deleting a curation:
<img width="1182" alt="" src="https://user-images.githubusercontent.com/549407/108121418-a5240900-7057-11eb-9f72-6cb32a05810f.png">

## QA

- Go to the sample engine or create some curations in the standalone UI
- Go to http://localhost:5601/xyz/app/enterprise_search/app_search/engines/national-parks-demo/curations
- [x] Confirm the curations in http://localhost:3002/as#/engines/national-parks-demo/curations load in the same
- [x] Confirm deleting a curation works as expected
- Route testing (goes to TODO views)
  - [x] Confirm that clicking the "Create new curation" button takes you to the correct route
  - [x] Confirm that clicking the Curation queries text links takes you to the correct curation ID route
  - [x] Confirm that clicking the edit pencil icon takes you to the correct curation ID route
- [optional pagination testing]
  - [x] Create 15 curations in the standalone UI, confirm that table pagination works as expected

### Checklist
